extensions/librdkafka/ConsumeKafka.h

```diff
@@ -126,7 +124,7 @@ class ConsumeKafka : public core::Processor {
  private:
   void create_topic_partition_list();
   void extend_config_from_dynamic_properties(const core::ProcessContext& context);
-  void configure_new_connection(const core::ProcessContext& context);
+  void configure_new_connection(core::ProcessContext* context);
```

Comment:
I wanted to align it with the parameters of the processor config utils functions, but it does make more sense to have all of them as references instead of pointers. Updated in 98ed8e2450436c2502a71f0bc99301fa4576db2c